packages/server-wallet/src/wallet/store.ts

```diff
@@ -94,6 +94,10 @@ export class Store {
     }
   }
 
+  async closeDatabaseConnection(): Promise<void> {
```

Comment:
This could probably just be named `destroy`:
- there may be multiple things that the store needs to clean up
- in a world where we have multiple implementations of a `Store` interface, we might not have a `Database`

Also, (extremely small) nit:
- database connections are ephemeral. They are created on demand, and the number can fluctuate depending on the `knex` config as well as both the wallet load (when not under stress, connections are closed after a timeout) and the DB load (the DB may forcibly close a connection for a variety of reasons)
- knex (well, `tarn`, under the hood) maintains a dynamic pool of connections
- `knex.destroy()` destroys the _pool_ of connections, and signals to `knex` not to start any new ones.

Reply:
https://github.com/statechannels/statechannels/pull/2602